CHANGES_NEXT_RELEASE

```diff
@@ -15,6 +15,9 @@
 - Fix: bug when tenants are over 44 bytes long (#2811)
 - Add: Support for null values in string filters (#2359)
 - Fix: Partial sanity check for custon url, only if the url contains no replacements using ${xxx} (#2280, #2279)
+- Fix: if entity::type is given but as an empty string, an error is returned (#1985)
+- Fix: admin requests with errors now return '400 Bad Request', not '200 OK'
```

Comment:
"adming request" could be a big obscure, and example would help. I mean:

"Fix: admin requests (e.g. GET /version) with errors now return '400 Bad Request', not '200 OK'"

Btw, issue number is missing at the end of the line. If the fix is not related with any issue, that's perfect (asking only to check if that is the case...)